cluster/apiserver/apiserver.yaml

```diff
@@ -72,3 +72,26 @@ items:
           image: ghcr.io/vega-project/vega-project/ccb-operator/apiserver:latest
           args:
           - --dry-run=false
+
+- apiVersion: v1
```

Comment:
Please open a new PR for the liveness probe changes. Also, this is wrong. Here you create a pod that runs an image that we don't need. We already have the deployments [here](https://github.com/vega-project/ccb-operator/blob/master/cluster/apiserver/apiserver.yaml#L50-L74)

Reply:
Oh, it is a separated PR, but you have the commits from the other PR as well.

Reply:
Yeah, I dont know why that happened...

Reply:
You did `git checkout -b liveness-prove` while you were in the `apiserver-tests` branch and you carried the commit from the `apiserver-tests` branch as well.

Reply:
Oh yes.. Totally forgot about that.